server/apps/distributed-app/docs/modules/ROOT/pages/configure/usersrepository.adoc

```diff
@@ -39,6 +39,12 @@ acting on the behalf of any user.
 | verifyFailureDelay
 | Delay after a failed authentication attempt with an invalid user name or password. Duration string defaulting to seconds, e.g. `2`, `2s`, `2000ms`. Default `0s` (disabled).
 
+| algorithm
+| use a specific hash algorithm to compute passwords, with optional mode `plain` (default) or `salted`; e.g. `SHA-512` (default),  `SHA-512/plain`, `SHA-512/salted`
```

Comment:
Opps, it is supposed to be just `SHA-512` (i.e. `plain`) for consistency. Since it was the default before, it should stay the same so it does not mess up peoples existing installations. Will fix it.
But I agree that it should be `SHA-512/slated` in the sample config files, for people starting a fresh installation.